test/models/reminder_mailing_volunteer_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class ReminderMailingVolunteerTest < ActiveSupport::TestCase
+  def setup
+    @superadmin = create :user
+    @volunteer = create :volunteer_with_user
+    @assignment_probation = create :assignment, period_start: 7.weeks.ago, volunteer: @volunteer
+  end
+
+  test 'template variables are substituted' do
+    reminder_mailing = ReminderMailing.new(kind: :probation_period, creator: @superadmin,
+      subject: 'hallo %{Anrede} %{Name}', reminder_mailing_volunteers: [@assignment_probation],
```

Comment:
RuboCop complains if this kind  `%{Anrede}` of form is selected for variables. It suggests this `%<Anrede>` instead.

Reply:
I have no clue what RuboCop expects me to do here, but as what he suggests to change is rubbish, I'm ignoring him.
This is because:

```ruby
2.4.2 (main):0 > 'hallo %{Anrede} %{Name}' % { Anrede: 'blabla', Name: 'Sonstwas' }
=> "hallo blabla Sonstwas"
2.4.2 (main):0 > 'hallo %<Anrede> %<Name>' % { Anrede: 'blabla', Name: 'Sonstwas' }
ArgumentError: invalid format character - %
2.4.2 (main):0 > format('hallo %<Anrede> %<Name>', Anrede: 'blabla', Name: 'Sonstwas')
ArgumentError: invalid format character - %
2.4.2 (main):0 > format('hallo %<Anrede> %<Name>', {Anrede: 'blabla', Name: 'Sonstwas'})
ArgumentError: invalid format character - %
2.4.2 (main):0 > format('hallo %<Anrede> %<Name>', {Anrede: 'blabla', Name: 'Sonstwas'})
ArgumentError: invalid format character - %
```

What rubocop suggests looks uglier, works uglier, .... well it just is 💩 

And the solution is simple:

```yaml
Style/FormatStringToken:
  EnforcedStyle: template
```

http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/FormatStringToken

Reply:
And the rubocop documentation doesn't say that this would be any worse than the other solution. Sometimes Rubocop could really give the reason why x should be enforced y ...

Reply:
It's based on the [Ruby style guide](https://github.com/bbatsov/ruby-style-guide#named-format-tokens):

> When using named format string tokens, favor `%<name>s` over `%{name}` because it encodes information about the type of the value.
>
>  ```ruby
>  # bad
>  format('Hello, %{name}', name: 'John')
>
>  # good
>  format('Hello, %<name>s', name: 'John')
>  ```

With `%<>` you also need to specify the type as a suffix (e.g. `%<Anrede>s`), which is why you got that error.

I'm happy with either style, as long as it's consistent ;-)